env/.env.test

```diff
@@ -0,0 +1,8 @@
+# Flask server configuration.
+FLASK_APP=todo_app/app
+FLASK_ENV=development
+
+# Change the following values for local development.
+SECRET_KEY=test_secret_key
+
+TRELLO_BOARD_ID=test_board_id
```

Comment:
It looks like `TRELLO_API_BASE_URL` is missing from this file. Also, when you're testing, the path that you use is `.env.test`, but this file is now in `env/.env.test`